bundles/model/org.openhab.model.script/src/org/openhab/model/script/actions/ScriptExecution.java

```diff
@@ -38,6 +41,11 @@
  * @author Kai Kreuzer
  * @since 0.9.0
  *
+ * Extension to also support reschedulable cron timers
+ * 
+ * @author Peter Broucke
+ * @since 0.9.1
```

Comment:
please enhance the general class description, add yourself as author directly beneath Kai and leave @since unchanged (it shall indicate when the class has been introduced.